packages/formation/sass/full.scss

```diff
@@ -2,31 +2,32 @@
  * Complete set of Formation styles
  */
 
-$fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
-$font-path: "~uswds/src/fonts";
-$image-path: "~uswds/src/img";
+$fa-font-path: '~@fortawesome/fontawesome-free/webfonts';
+$font-path: '~uswds/src/fonts';
+$image-path: '~uswds/src/img';
 
-@import "core";
-@import "modules/m-alert";
-@import "modules/m-breadcrumbs";
-@import "modules/m-table";
-@import "modules/m-button";
-@import "modules/m-form-elements";
-@import "modules/m-search";
-@import "modules/m-loading-indicator";
-@import "modules/m-print";
-@import "modules/m-dropdown";
-@import "modules/m-nav-sidebar";
-@import "modules/m-overlay";
-@import "modules/m-additional-info";
-@import "mobile-typography";
-@import "shame";
-@import "modules/m-modal";
-@import "modules/m-progress-bar";
-@import "modules/va-pagination";
-@import "modules/va-tabs";
-@import "modules/m-process-list";
-@import "icons";
-@import "modules/m-megamenu";
-@import "modules/m-hub-page-link-list";
-@import "modules/m-promo-banner";
+@import 'core';
+@import 'modules/m-alert';
+@import 'modules/m-breadcrumbs';
+@import 'modules/m-table';
+@import 'modules/m-button';
+@import 'modules/m-form-elements';
+@import 'modules/m-search';
+@import 'modules/m-loading-indicator';
+@import 'modules/m-print';
+@import 'modules/m-dropdown';
+@import 'modules/m-nav-sidebar';
+@import 'modules/m-overlay';
+@import 'modules/m-additional-info';
+@import 'mobile-typography';
+@import 'shame';
+@import 'modules/m-modal';
+@import 'modules/m-progress-bar';
+@import 'modules/va-pagination';
+@import 'modules/va-tabs';
+@import 'modules/m-process-list';
+@import 'icons';
+@import 'modules/m-megamenu';
+@import 'modules/m-hub-page-link-list';
+@import 'modules/m-promo-banner';
+@import 'modules/m-responsive-tables';
```

Comment:
I put this in here because everything else was in here :shrug:

Reply:
Yeah, as long as this is the pattern we've got for now, that's good. We'll probably reassess this approach later.